### PR TITLE
add current actual google build admins to build-admins

### PR DESCRIPTION
**Pull request by BenTheElder**
unclear which of the other entries should be removed

See: https://github.com/kubernetes/sig-release/issues/1150

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -208,6 +208,10 @@ teams:
               the Kubernetes project. This team is a notification-only group,
               which includes the SIG Release Chairs for continuity.
             members:
+            - amwat
```
Comment (MushuEE) [inline config/kubernetes/sig-release/teams.yaml:211]: Also maybe alpha? Unless we are taking over the whole block

Comment (spiffxp) [reply config/kubernetes/sig-release/teams.yaml:211]: I think CI might fail on non-alpha anyway

Comment (BenTheElder) [reply config/kubernetes/sig-release/teams.yaml:211]: I think maybe the others should be removed but not clear.
humans alpha sorting things by hand is just the worst.

**Comment by BenTheElder** [comment]
>   config_test.go:251: The team build-admins in org Kubernetes has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: spiffxp

... really? is this a github limitation or self-imposed?

>    config_test.go:251: The team build-admins in org Kubernetes has an unsorted list of members

fun

**Comment by spiffxp** [comment]
>... really? is this a github limitation or self-imposed?

github

**Pull request merged**